### PR TITLE
Fix WP core checkbox styles introduce odd visual effect.

### PR DESCRIPTION
**Pull request by benbowler**
## Summary

Addresses issue #2722

## Relevant technical choices

I used `opacity: 0 !important` because this is what is causing the error, in certain states, MD set's the opacity to 0.7. By overriding this we can make sure it stays hidden.

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/sass/vendor/_mdc-switch.scss [hunk 1]
```diff
@@ -29,4 +29,10 @@
 	+ label {
 		margin-left: 10px;
 	}
+
+	// Prevent the background checkbox from becoming visible when a switch
+	// loads or is set in the disabled state.
+	.mdc-switch__native-control::before {
```
Comment (aaemnnosttv) [inline assets/sass/vendor/_mdc-switch.scss:35]: It looks like we already have some styles in place here for "Overriding WordPress load-styles.php" above on the native control. Can we merge this into that selector above? e.g. `&::before`

**Review by aaemnnosttv — COMMENTED** [review]
Thanks @benbowler ! Looks good, just one question about how we integrate this with the existing styles.

**Comment by benbowler** [comment]
Thanks @aaemnnosttv, I've merged that in.

**Review by aaemnnosttv — APPROVED** [review]
Great, thanks!

**Pull request merged**